backend/controllers/createLinkedDpsUser.test.js

```diff
@@ -0,0 +1,384 @@
+const { createLinkedDpsUserFactory } = require('./createLinkedDpsUser')
+
+describe('create linked user factory', () => {
+  const getCaseloads = jest.fn()
+  const createLinkedAdminUser = jest.fn()
+  const createLinkedLsaUser = jest.fn()
+  const createLinkedGeneralUser = jest.fn()
+  const searchUser = jest.fn()
+
+  const createLinkedDpsUser = createLinkedDpsUserFactory(
+    getCaseloads,
+    createLinkedAdminUser,
+    createLinkedLsaUser,
+    createLinkedGeneralUser,
+    searchUser,
+    '/create-linked-dps-user',
+    '/create-dps-user',
+    '/create-user',
+    '/manage-dps-users',
+  )
+
+  describe('index', () => {
+    it('should call create-user index scenario 1', async () => {
```

Comment:
What are scenarios 1, 2 & 3? Please include a description of what the scenario under test is i.e.
`it('should call create-user index when <insert scenario description here>', async () => {`